site/en/blog/background-fetch/index.md

```diff
@@ -469,26 +466,10 @@ addEventListener('backgroundfetchclick', (event) => {
 });
 ```
 
-## Origin trial
-
-To get access to this new API on your site, please [sign up](http://bit.ly/OriginTrialSignup) for
-the "Background Fetch API" Origin Trial. If you just want to try it out locally, the API can be
-enabled on the command line:
-
-```shell
-chrome --enable-blink-features=BackgroundFetch
-```
-
-Passing this flag on the command line enables the API globally in Chrome for the current session.
-
-If you give this API a try, please let us know what you think! Please direct feedback on the API
-shape to [the specification repository](https://github.com/WICG/background-fetch/issues), and report
-implementation bugs to [the BackgroundFetch Blink
-component](https://bugs.chromium.org/p/chromium/issues/entry?template=Defect+report+from+developer&components=Blink%3EBackgroundFetch).
-
 ## Additional resources
 
 * [Explainer](https://github.com/WICG/background-fetch/blob/main/README.md)
 * [Specification](https://wicg.github.io/background-fetch/)
 * [chromestatus.com entry](https://www.chromestatus.com/features/5712608971718656)
 
+_Correction: A previous version of this article incorrectly referred to Background Fetch as being a "web standard". The API is not currently on the standards track, the [specification](https://wicg.github.io/background-fetch/) can be found in WICG as a Draft Community Group Report._
```

Comment:
Do we usually put changelogs in the article like this?

Reply:
I am suggesting that we do in the guidelines I'm creating, where there is inaccuracy fixed (in common with most editorial guidelines, which generally discourage stealth edits).

Reply:
Huh, so I'm currently updating articles for a couple of APIs that have undergone significant changes. Doing inline changelogs will double the length of the article and make it pretty hard to read 😢

Reply:
Just put a note at the end like I did? If people want to see all the details they can poke around GitHub.

Reply:
The aim is to alert people that major things have changed - as people link to our articles and then we make them essentially say something different. It's one of the issues of people using blog posts as docs, as the expectations are wrong, but here we are.

Reply:
For comparison, I'm updating an article that Sam wrote, which is now incorrect because the API has since changed. I think it would be wrong for me to claim "a previous version of this article incorrectly claimed…", and fairer to say "A previous version of this article referred to an older version of the API" (although I'm not sure how useful that is in general)

Reply:
Yeah, I think your suggestion is fine in that case. I've definitely been in the position of quoting from an article that has then been changed, making my post look weird :D so a note like that stops someone who referenced us looking like they were talking nonsense. It's also generally seen as good practice and I'm trying to make our editorial policies a bit more robust.